test/fail/ok/suggest-local-type-ai.tc.ok

```diff
@@ -0,0 +1,8 @@
+suggest-local-type-ai.mo:3.13-3.15: type error [M0029], unbound type Fo in type environment:
+  {
+    type Foo = Int;
+    type UpgradeOptions = {wasm_memory_persistence : ?WasmMemoryPersistence};
+    type WasmMemoryPersistence = {#Keep; #Replace};
```

Comment:
Hmm, these should not be leaking out. Will prefix with @ like other internals.

Reply:
Done https://github.com/dfinity/motoko/pull/4720/commits/6b753868efcd92f265c1129d34dcd8db7cefc5c1